xchange-coinbene/src/main/java/org/knowm/xchange/coinbene/CoinbeneExchange.java

```diff
@@ -46,12 +46,12 @@ public ExchangeSpecification getDefaultExchangeSpecification() {
   }
 
   @Override
-  public void remoteInit() {
+  public void remoteInit() throws IOException, ExchangeException {
 
     try {
       exchangeMetaData = ((CoinbeneMarketDataService) marketDataService).getMetadata();
     } catch (IOException e) {
-      LOG.error("Retrieving MetaData for Coinbene failed", e);
+      throw new IOException("Retrieving MetaData for Coinbene failed", e);
```

Comment:
remoteInit() throws IOException, so you don't have to catch it and rethrow it. Just take the try catch away completely.